packages/cli/configuration/src/generators-yml/schemas/APIConfigurationV2Schema.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from "zod";
+import { RawSchemas } from "@fern-api/fern-definition-schema";
+
+export const OpenAPISpecSchema = z.strictObject({
+    openapi: z.string(),
```

Comment:
nit should this just be `api` with the `specs` block and such? just since this is "v2"?